core/tests/protractor.conf.js

```diff
@@ -119,6 +119,10 @@ exports.config = {
       'protractor/subscriptionsFlow.js'
     ],
 
+    translatorFeatures: [
```

Comment:
Hi @DubeySandeep, just a doubt -- where is this suit located?

Reply:
Oops! I forgot to remove this!
Thanks!

Reply:
Done!